### ISSUE TITLE
Gate of the Fold sending player back to the initial dimension

### ISSUE THREAD

**Issue by PlayerDot** [WayofTime/BloodMagic#1620]
#### Issue Description:
I initially encountered this in SevTech and a couple of other modpacks, so I made a world with just BM installed and cheated myself the ritual ingredients to test it out myself and it still happens.
If you take too long to load the location you are travelling to, the delay of the portal between uses will have reset by the time you load in and can move around, which will cause you to activate the target portal and be sent to the initial portal. This can cause you to rubberband between different locations for several minutes before you get out. This is especially critical when travelling between dimensions, because it takes a substantial amount of time to load the dimensions. 

#### What happens:
When teleporting from one location to another, you appear in the middle of the portal frame and the portals keep sending you back and forth between each other if you take too long to load the location.

#### What you expected to happen:
Not spawning immediately inside the portal which would cause me to teleport again. Not being able to reuse the portal until I've moved a few blocks outside it.

#### Steps to reproduce:

1.  Place Gate of Fold in one dimension
2.  Connect Gate of Fold in another dimension (or same dimension, but really far away)
3.  Take too long to load or move out of the portal frame
4.  Congratulations, you are now stuck teleporting between portals over and over again.

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: 1.12.2-2.2.12-97
- Minecraft:  1.12.12
- Forge: 14.23.4.2707

**Comment by keraldi** [comment]
Weird. Because I'm pretty sure I remember "fixing" that a long time ago...

Ah, here it is: https://github.com/WayofTime/BloodMagic/pull/1387
You should probably update Blood Magic. Your version is over a year old.

You may be forgiven. It's not in the changelog. But update Blood Magic and test it again.

Close the issue if it's resolved then.

**Comment by PlayerDot** [comment]
> Weird. Because I'm pretty sure I remember "fixing" that a long time ago...
> 
> Ah, here it is: #1387
> You should probably update Blood Magic. Your version is over a year old.
> 
> You may be forgiven. It's not in the changelog. But update Blood Magic and test it again.
> 
> Close the issue if it's resolved then.

Yep, sorry about that. I didn't realize the version I had from this modpack was that old. They never bother updating. It works fine now.
